Auxillary_files/MIP_grids.json

```diff
@@ -1,161 +1,173 @@
-      "standard_name": "",
-      "type": "integer",
-      "units": "1"
-    },
-    "vertices": {
-      "axis": "",
-      "long_name": "",
-      "out_name": "",
-      "standard_name": "",
-      "type": "",
-      "units": ""
-    },
-    "x": {
-      "axis": "X",
-      "long_name": "x coordinate of projection",
-      "out_name": "",
-      "standard_name": "projection_x_coordinate",
-      "type": "double",
-      "units": "m"
-    },
-    "x_deg": {
-      "axis": "X",
-      "long_name": "x coordinate of projection",
-      "out_name": "x",
-      "standard_name": "projection_x_coordinate",
-      "type": "double",
-      "units": "degrees"
-    },
-    "y": {
-      "axis": "Y",
-      "long_name": "y coordinate of projection",
-      "out_name": "",
-      "standard_name": "projection_y_coordinate",
-      "type": "double",
-      "units": "m"
-    },
-    "y_deg": {
-      "axis": "Y",
-      "long_name": "y coordinate of projection",
-      "out_name": "y",
-      "standard_name": "projection_y_coordinate",
-      "type": "double",
-      "units": "degrees"
-    }
-  },
-  "mapping_entry": {
-    "sample_user_mapping": {
-      "coordinates": "rlon rlat",
-      "parameter1": "false_easting",
-      "parameter2": "false_northing"
-    }
-  },
-  "variable_entry": {
-    "latitude": {
-      "dimensions": "longitude latitude",
-      "long_name": "latitude",
-      "out_name": "latitude",
-      "standard_name": "latitude",
-      "type": "double",
-      "units": "degrees_north",
-      "valid_max": "90.0",
-      "valid_min": "-90.0"
-    },
-    "longitude": {
-      "dimensions": "longitude latitude",
-      "long_name": "longitude",
-      "out_name": "longitude",
-      "standard_name": "longitude",
-      "type": "double",
-      "units": "degrees_east",
-      "valid_max": "360.0",
-      "valid_min": "0.0"
-    },
-    "vertices_latitude": {
-      "dimensions": "vertices longitude latitude",
-      "long_name": "",
-      "out_name": "vertices_latitude",
-      "standard_name": "",
-      "type": "double",
-      "units": "degrees_north",
-      "valid_max": "90.0",
-      "valid_min": "-90.0"
-    },
-    "vertices_longitude": {
-      "dimensions": "vertices longitude latitude",
-      "long_name": "",
-      "out_name": "vertices_longitude",
-      "standard_name": "",
-      "type": "double",
-      "units": "degrees_east",
-      "valid_max": "360.0",
-      "valid_min": "0.0"
+    "version_metadata": {
+        "version_tag": "v6.5.0.0",
+        "checksum": "md5: fece7447818cb5fc60fa4478c4d38d65",
+        "Auxillary_files/MIP_grids_modified": "Thu Nov 16 14:33:58 2023 +0000",
+        "Auxillary_files/MIP_grids_note": "Relocating auxillary files following review",
+        "commit": "68524fc1ae4fb2b971cf12f8f30e074709cfb1e9",
+        "previous_commit": "",
+        "author": "Matthew Mizielinski",
+        "institution_id": "MOHC",
+        "CV_collection_version": "6.5.0.0",
+        "specs_doc": "v6.5.0"
```

Comment:
Just a note that the specifications doc only exists for CMIP6 with a version 6.2.7 - see https://docs.google.com/document/d/1h0r8RZr_f3-8egBMMh7aqLwy3snpD6_MrDz1q8n5XUk/edit

Reply:
It made sense that a new specs document will be stood up in the future and that these should reflect that. Even if the new document just states 'please refer to 6.2.7' . @matthew-mizielinski 

I can easily revert the metadata to that one if it is simpler?

Reply:
While the document is fairly straightforward there are a number of links to CMIP6.  The separation of mip tables and CVs starts becoming awkward here, in that these numbers are really referring to stuff on the CMIP6Plus_CVs repository, not here. Not sure this is worth worrying about now, but we should discuss this again